packages/samples/headless-ssr/package-lock.json

```diff
@@ -0,0 +1,3610 @@
+{
```

Comment:
You should definitely not have a lock file here.

Reply:
Just gotta add this project to the `workspaces` in the root `package.json`